monai/apps/pathology/datasets.py

```diff
@@ -160,3 +166,130 @@ def __init__(
             progress=progress,
             shuffle=False,
         )
+
+
+class MaskedInferenceWSIDataset(Dataset):
+    """
+    This dataset load the provided tissue masks at an arbitrary resolution level,
+    and extract patches based on that mask from the associated whole slide image.
```

Comment:
looks like `mask_locations` is not used in data loading, could you clarify?

Reply:
`image_location` is being calculated based on `mask_location`, it is also returned by dataset since inference output is in the size of `mask` and not `image`: https://github.com/Project-MONAI/MONAI/blob/6f9edbe0e8873425f9abbd97422e7d7a6841b0a1/monai/apps/pathology/datasets.py#L285

Reply:
Are we good, here?

Reply:
yes this one is fine